atd-vzd/migrations/default/1715094801442_add_import_log_table/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE cris_import_log (
+    id SERIAL PRIMARY KEY,
+    object_path TEXT NOT NULL,
+    object_name TEXT NOT NULL,
+    first_seen TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    import_attempted BOOLEAN NOT NULL DEFAULT FALSE,
+    import_successful BOOLEAN NOT NULL DEFAULT FALSE,
+    import_time TIMESTAMPTZ DEFAULT NULL
```

Comment:
how do you feel about `created_at` and `updated_at` (or `completed_at`) for these timestamp column names? 

do you think it would it be worth going ahead and creating a compound index on (`object_path`, `object_name`)? perhaps that's overkill since we're talking about 365 rows/year on average. 

similarly, and i guess this would accomplish the indexing automatically, do we want to add a unique constraint against those two columns?

Reply:
Yep. The audit field names are a billion times better. They are adopted. ✨ 

Regarding additional indexes & constraints, given the changes since this comment was written, I think we're where we want to be. The log records a record for each processing attempt, so there is no longer an expectation for unicity of the object path + name.